apisix/cli/ngx_tpl.lua

```diff
@@ -660,6 +664,10 @@ http {
             set $dubbo_method                '';
             {% end %}
 
+            {% if enabled_plugins["saml-auth"] then %}
+            set $saml_data_dir               {*apisix_lua_home*}/deps/share/lua/5.1/resty/saml;
```

Comment:
Look like we can put the generated saml_data_dir in the Lua code? So there is no need to use extra variable.

Reply:
It depends on `apisix_lua_home`, how to determine it in lua code?